mantidproject/mantid
# Add canvas color to figure axes options

**Description of work.**
This PR adds a canvas color option to the figure axes options.

**Report to:** SANS group / Steve King

**To test:**

1. Load some data
2. Plot the data
3. Go to figure options
4. There is a new canvas color setting
5. Change it and press apply - the plot should update
6. Likewise this should work for tiled plotting. 

Fixes #28607.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
